tutorials/quick-kubernetes.md

```diff
@@ -0,0 +1,16 @@
+# Quick kubernetes setup
+
+This tutorial assumes you've already installed CRI-O and have started it. If you have not, start [here](setup.md).
+
+You can run a local version of kubernetes with CRI-O using `local-up-cluster.sh`:
+
+1. Clone the [kubernetes repository](https://github.com/kubernetes/kubernetes)
+1. From the kubernetes project directory, run:
```

Comment:
```suggestion
2. From the kubernetes project directory, run:
```

Reply:
I think it's all 1. so Markdown is in charge of the numbering